### PR TITLE
Bugfix/#6370

### PR DESCRIPTION
**Pull request by KKatiaa**
There are no indents from the right edge of the page [#6351](https://github.com/ita-social-projects/GreenCity/issues/6351)
The "About service" button does not work. UBS footer. [#6345](https://github.com/ita-social-projects/GreenCity/issues/6345)
When adding 'Add service', the number in the 'Basic cost' column is not displayed correctly [#6370](https://github.com/ita-social-projects/GreenCity/issues/6370)

### CODE REVIEW

File: src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts [hunk 1]
```diff
@@ -288,11 +288,15 @@ export class AddNewHabitComponent implements OnInit {
     this.friendsIdsList = this.userFriendsService.addedFriends?.map((friend) => friend.id);
     const defailtItemsIds = this.standartShopList.filter((item) => item.selected === true).map((item) => item.id);
     const habitAssignProperties: HabitAssignPropertiesDto = { defaultShoppingListItems: defailtItemsIds, duration: this.newDuration };
+    console.log(habitAssignProperties, defailtItemsIds, this.standartShopList, 'habitAssignProperties');
```
Comment (hnativlyubomyr) [inline src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts]: remove console.log

File: src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts [hunk 2]
```diff
@@ -288,11 +288,15 @@ export class AddNewHabitComponent implements OnInit {
...
     this.habitAssignService
       .assignCustomHabit(this.habitId, this.friendsIdsList, habitAssignProperties)
       .pipe(take(1))
       .subscribe(() => {
-        this.customShopList.length > 0 ? this.addCustomHabitItems() : this.afterHabitWasChanged('habitAdded');
+        if (this.customShopList.length > 0) {
```
Comment (hnativlyubomyr) [inline src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts]: if (this.customShopList.length)

File: src/app/shared/header/header.component.ts [hunk 3]
```diff
@@ -92,6 +92,7 @@ export class HeaderComponent implements OnInit, OnDestroy {
     this.imgAlt = this.isUBS ? 'Image ubs logo' : 'Image green city logo';
     this.localeStorageService.setUbsRegistration(this.isUBS);
     this.toggleHeader();
+    console.log(this.navLinks, 'navLinks');
```
Comment (hnativlyubomyr) [inline src/app/shared/header/header.component.ts]: the same

File: src/app/shared/ubs-footer/ubs-footer.component.ts [hunk 4]
```diff
@@ -22,11 +22,16 @@ export class UbsFooterComponent {
 
   constructor(private dialog: MatDialog) {}
 
+  ngOnInit() {
+    console.log(this.ubsNavLinks, 'ubsNavLinks');
```
Comment (hnativlyubomyr) [inline src/app/shared/ubs-footer/ubs-footer.component.ts]: remove it

File: src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts (hunk 1 above)
Comment (KKatiaa) [reply src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts]: removed all console.logs

**Review by NadiiaStoiko — APPROVED** [review]

**Review by OksanaFilipchuk — APPROVED** [review]

**Review by OksanaFilipchuk — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**